frontend/app/.eslintrc.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
       process.env.NODE_ENV === 'development' ? 'warn' : 'error',
       {
         ignoreNodes: ['md-icon', 'v-icon'],
-        ignorePattern: '^[-#:()&/+]+$',
+        ignorePattern: '^[-#:()&/+=!]+$',
```

Comment:
should I make this just allow all symbols?

Reply:
No need, just keep it strictly to the ones we use.